test/handle_tests.jl

```diff
@@ -0,0 +1,15 @@
+println("Test empty isempty")
+h1 = Sundials.Handle(Sundials.CVodeCreate(Sundials.CV_BDF, Sundials.CV_NEWTON))
+h2 = h1
+
+@test !isempty(h1)
+@test !isempty(h2)
+
+empty!(h1)
+
+@test isempty(h1)
+@test isempty(h2)
+
+println("Handle pointing to NULL")
```

Comment:
This is as well `@testset` candidate. I would avoid to much explicit printing in tests. If the test fails, then `@testset` macro is supposed to show what and where have failed. If the tests are fine, this line doesn't provide useful info.